src/lib/platform/OSXScreen.mm

```diff
@@ -790,6 +790,10 @@
 void
 OSXScreen::enter()
 {
+    // Mark as on screen so other events like are handled as on screen.
+    // Mitigates https://github.com/input-leap/input-leap/issues/1043 from the bogus movement check
+	m_isOnScreen = true;
```

Comment:
m_isOnScreen below should be deleted then.

Reply:
You are absolutely right. I've corrected the pull request.